diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59afb51..6bad1e6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -154,8 +154,8 @@ function(build_Qt6)
     # https://bugreports.qt.io/browse/QTBUG-114706
     add_library(Qt::Core ALIAS Qt6::Core)
     add_library(Qt::Gui ALIAS Qt6::Gui)
-    add_library(Qt::OpenGL ALIAS Qt6::OpenGL)
-    add_library(Qt::Network ALIAS Qt6::Network)
+    # add_library(Qt::OpenGL ALIAS Qt6::OpenGL)
+    # add_library(Qt::Network ALIAS Qt6::Network)
     if(Qt6_VERSION_MINOR GREATER 6)
         add_library(Qt::PlatformModuleInternal ALIAS Qt6::PlatformModuleInternal)
     endif()
